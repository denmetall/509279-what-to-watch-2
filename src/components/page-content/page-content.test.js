import React from 'react';
import renderer from 'react-test-renderer';
import PageContent from "./page-content.jsx";
import filmsMock from '../../mocks/films';
import {HashRouter as Router} from 'react-router-dom';

it(`Components PageContent renders correctly`, () => {
  const tree = renderer
    .create(<Router>
      <PageContent films={filmsMock} onClickTitleHandler={jest.fn()}/>
    </Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
