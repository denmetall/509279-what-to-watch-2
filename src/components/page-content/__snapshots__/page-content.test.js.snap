// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components PageContent renders correctly 1`] = `
<div
  className="page-content"
>
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <ul
      className="catalog__genres-list"
    >
      <li
        className="catalog__genres-item catalog__genres-item--active"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          All genres
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Comedies
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Crime
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Documentary
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Dramas
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Horror
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Kids & Family
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Romance
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Sci-Fi
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Thrillers
        </a>
      </li>
    </ul>
    <div
      className="catalog__movies-list"
    >
      <article
        className="small-movie-card catalog__movies-card"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <div
          className="small-movie-card__image"
        >
          <video
            height="175"
            muted={true}
            poster="img/johnny-english.jpg"
            width="280"
          >
            <source
              src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
            />
          </video>
        </div>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="#/film/1"
            onClick={[Function]}
          >
            Drive
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <div
          className="small-movie-card__image"
        >
          <video
            height="175"
            muted={true}
            poster="img/macbeth.jpg"
            width="280"
          >
            <source
              src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
            />
          </video>
        </div>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="#/film/2"
            onClick={[Function]}
          >
            Astral
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <div
          className="small-movie-card__image"
        >
          <video
            height="175"
            muted={true}
            poster="img/johnny-english.jpg"
            width="280"
          >
            <source
              src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
            />
          </video>
        </div>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="#/film/3"
            onClick={[Function]}
          >
            Taxi
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <div
          className="small-movie-card__image"
        >
          <video
            height="175"
            muted={true}
            poster="img/macbeth.jpg"
            width="280"
          >
            <source
              src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
            />
          </video>
        </div>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="#/film/4"
            onClick={[Function]}
          >
            Resident evil
          </a>
        </h3>
      </article>
      <article
        className="small-movie-card catalog__movies-card"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <div
          className="small-movie-card__image"
        >
          <video
            height="175"
            muted={true}
            poster="img/mindhunter.jpg"
            width="280"
          >
            <source
              src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
            />
          </video>
        </div>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="#/film/5"
            onClick={[Function]}
          >
            Brother
          </a>
        </h3>
      </article>
    </div>
    <div
      className="catalog__more"
    >
      <button
        className="catalog__button"
        type="button"
      >
        Show more
      </button>
    </div>
  </section>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="main.html"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
